bot/cogs/reddit.py

```diff
@@ -63,23 +74,18 @@ def __init__(self, bot: Bot):
         log.debug(f"Invalid response from: {url} - status code {response.status}, mimetype {response.content_type}")
         return list()  # Failed to get appropriate response within allowed number of retries.
 
-    async def send_top_posts(
-        self, channel: TextChannel, subreddit: Subreddit, content: str = None, time: str = "all"
-    ) -> Message:
-        """Create an embed for the top posts, then send it in a given TextChannel."""
+    async def get_top_posts(self, subreddit: Subreddit, time: str = "all") -> Embed:
+        """Returns an embed for the top posts of the given subreddit."""
```

Comment:
```suggestion
        """Get the top 5 posts for a subreddit within a specified timeframe."""
```
Couple of things here:
* Use imperative mood in docstrings, e.g. `Return an embed`, not `Returns an embed`.
* Don't describe the return type, explain what the method _does_. We have return type annotations to explain what it returns.
* Be specific. Explaining how the `time` parameter works, for example, seems like useful API documentation, and in my opinion you should spend a few more lines doing just that beyond what I suggested here.